tests/Fixtures/Integration/misc/phpdoc_to_comment,phpdoc_var_without_name.test

```diff
@@ -1,26 +0,0 @@
---TEST--
-Integration of fixers: phpdoc_to_comment,phpdoc_var_without_name.
```

Comment:
It's a useless test that tests nothing, both fixers have no effect on the given code, as is already proven by their unit tests. There is no interaction going on between these fixers, so this test serves no use.

"Do not remove tests" sounds like a rule without reason. Can you tell me why this test should exist?

Reply:
for dropping test you assume it's pointless
for keeping it, we ensure that when test was introduced to ensure something works, we don't break that by accident, because we happily drop testing that.

I prefer to have 10 tests that test nothing than drop one test that one only thought it doesn't test a thing.

Test here was added as proof that false-positive is not happening anymore, so prove that bug we had was fixed. That means originally, this code was wrongly changed.
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2191